packages/database/src/migrations/20210713071004-AddFluTrackerNonFirstNationWithIliMapOverlay-modifies-data.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const REPORT_CODE = 'AU_FLUTRACKING_Percentage_Non_First_Nations_ILI';
+
+const REPORT = {
+  id: generateId(),
+  code: REPORT_CODE,
+  config: {
+    fetch: {
+      aggregations: [
+        'FINAL_EACH_DAY',
+        {
+          type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+          config: {
+            dataSourceEntityType: 'sub_district',
+            aggregationEntityType: 'district',
+          },
+        },
+        'MOST_RECENT',
+      ],
+      dataElements: ['FWV_LGA_004b', 'FWV_LGA_003b'],
+    },
+    transform: [
+      'keyValueByDataElementName',
+      {
+        transform: 'aggregate',
+        organisationUnit: 'group',
+        '...': 'last',
+      },
+      {
+        transform: 'select',
+        "'organisationUnitCode'": '$row.organisationUnit',
+        "'% of participants with influenza like illness (ILI)'":
+          'divide($row.FWV_LGA_004b, $row.FWV_LGA_003b)',
+        "'Participants'": '$row.FWV_LGA_003b',
+        "'Participants with influenza like illness (ILI)'": '$row.FWV_LGA_004b',
+      },
+    ],
+  },
+};
+
+const MAP_OVERLAY = {
+  id: REPORT_CODE,
+  name: '% of non-First Nation participants with influenza like illness (ILI)',
+  userGroup: 'Public',
+  dataElementCode: '% of participants with influenza like illness (ILI)',
+  isDataRegional: true,
+  measureBuilder: 'useReportServer',
+  measureBuilderConfig: {
+    dataSourceType: 'custom',
+    reportCode: REPORT_CODE,
+  },
+  presentationOptions: {
+    scaleType: 'performanceDesc',
+    displayType: 'shaded-spectrum',
+    measureLevel: 'District',
+    valueType: 'percentage',
+    scaleBounds: {
+      left: {
+        max: 0,
+        min: 0,
+      },
+      right: {
+        max: 0.035,
+        min: 0.035,
+      },
+    },
+    measureConfig: {
+      Participants: {
+        type: 'popup-only',
+        hideFromLegend: true,
+      },
+      'Participants with influenza like illness (ILI)': {
+        type: 'popup-only',
+        hideFromLegend: true,
+      },
```

Comment:
```suggestion
      $all {
        type: 'popup-only',
        hideFromLegend: true,
      }

```
We can use $all config

Reply:
Ahh good call!